distribution/docs/src/main/resources/content/_using/view-histogram.adoc

```diff
@@ -6,8 +6,9 @@
 :order: 02
 
 == {title}
+((({title})))(((Histogram)))
```

Comment:
❔ Does this create separate items in the index?

Reply:
It does